docs/_toc.yml

```diff
@@ -20,13 +28,19 @@ parts:
       file: model_embeddings
     - title: Finetuning
       file: model_finetuning
-- caption: Reference Documentation
+- caption: Tutorials
   chapters:
-    - title: Changelog
-      file: changelog
+    - title: Generative AI for pixel reconstruction
+      file: 240105-clay-v0-reconstruction
+    - title: Create location embeddings
+      file: 240105-location-embeddings
+    - title: Interpolating images in embedding space
+      file: 240105-clay-v0-interpolation
```

Comment:
Probably should change these filenames, otherwise the URL comes up as `https://clay-foundation.github.io/model/240105-clay-v0-reconstruction.html` which doesn't look nice.

Reply:
Done in 9926919ef369634ff87bddcf4ad3e4f979326c0c